Zig build fails for root project

I noticed that if you try to run `zig build` for the root project it fails with the following message:

```
install
└─ install godot
   └─ zig build-lib godot Debug native 1 errors
src\api\Godot.zig:1:1: error: file exists in multiple modules
src\api\Godot.zig:1:1: note: root of module godot
src\api\Godot.zig:1:1: note: root of module root
src\api\Variant.zig:2:23: note: imported from module root
const Godot = @import("Godot.zig");
                      ^~~~~~~~~~~
error: the following command failed with 1 compilation errors:
C:\Users\Simon\scoop\persist\zvm\data\0.13.0\zig.exe build-lib -ODebug --dep godot --dep GodotCore --dep build_options -Mroot=C:\Users\Simon\Projects\godot-zig\src\api\Godot.zig -ODebug -I C:\Users\Simon\Projects\godot-zig\.zig-cache\o\9359d720b5f10679c03a8b8ac0c847ef\.zig-cache\tmp\dbadba7713a947da -I C:\Users\Simon\Projects\godot-zig\.zig-cache\tmp\a48f406bc2d4126a --dep GodotCore --dep build_options=build_options0 -Mgodot=C:\Users\Simon\Projects\godot-zig\src\api\Godot.zig -ODebug -I C:\Users\Simon\Projects\godot-zig\.zig-cache\tmp\a48f406bc2d4126a --dep godot -MGodotCore=C:\Users\Simon\Projects\godot-zig\.zig-cache\o\9359d720b5f10679c03a8b8ac0c847ef\.zig-cache\tmp\dbadba7713a947da\GodotCore.zig -Mbuild_options=C:\Users\Simon\Projects\godot-zig\.zig-cache\c\15052cdb85cbac4cb6d3d390991e1f18\options.zig -Mbuild_options0=C:\Users\Simon\Projects\godot-zig\.zig-cache\c\15052cdb85cbac4cb6d3d390991e1f18\options.zig -femit-implib --cache-dir C:\Users\Simon\Projects\godot-zig\.zig-cache --global-cache-dir C:\Users\Simon\AppData\Local\zig --name godot -dynamic --listen=-
Build Summary: 9/12 steps succeeded; 1 failed (disable with --summary none)
install transitive failure
└─ install godot transitive failure
   └─ zig build-lib godot Debug native 1 errors
error: the following build command failed with exit code 1:
C:\Users\Simon\Projects\godot-zig\.zig-cache\o\9aa7499500dd251ef6d49b82be05e849\build.exe C:\Users\Simon\scoop\persist\zvm\data\0.13.0\zig.exe C:\Users\Simon\Projects\godot-zig C:\Users\Simon\Projects\godot-zig\.zig-cache C:\Users\Simon\AppData\Local\zig --seed 0x2cd2bd42 -Zc1abe64dfe316588
```

It doesn't appear to affect the example projects. 

Any thoughts @r-bar?

--- a/build.zig
+++ b/build.zig
@@ -13,6 +13,9 @@ pub fn build(b: *std.Build) !void {
         "headers",
         "Where to source Godot header files. [options: GENERATED, VENDORED, <dir_path>] [default: GENERATED]",
     ) orelse "GENERATED";
+    const build_options = b.addOptions();
+    build_options.addOption([]const u8, "precision", precision);
+    build_options.addOption([]const u8, "headers", headers);
 
     const gdextension = build_gdextension(b, godot_path, headers);
     const binding_generator_step = b.step("binding_generator", "Build the binding_generator program");
@@ -30,25 +33,15 @@ pub fn build(b: *std.Build) !void {
 
     const bindgen = build_bindgen(b, gdextension.iface_headers.dirname(), binding_generator, precision, arch);
 
-    const lib = b.addSharedLibrary(.{
-        .name = "godot",
-        .root_source_file = b.path(b.pathJoin(&.{ "src", "api", "Godot.zig" })),
-        .target = target,
-        .optimize = optimize,
-    });
     const godot_module = b.addModule("godot", .{
         .root_source_file = b.path(b.pathJoin(&.{ "src", "api", "Godot.zig" })),
         .target = target,
         .optimize = optimize,
     });
-    godot_module.addAnonymousImport("GodotCore", .{
-        .root_source_file = bindgen.godot_core_path,
-        .link_libc = true,
-        .optimize = optimize,
-        .target = target,
-    });
+    godot_module.addOptions("build_options", build_options);
     godot_module.addIncludePath(bindgen.output_path);
     godot_module.addIncludePath(gdextension.iface_headers.dirname());
+
     const godot_core_module = b.addModule("GodotCore", .{
         .root_source_file = bindgen.godot_core_path,
         .target = target,
@@ -56,17 +49,8 @@ pub fn build(b: *std.Build) !void {
     });
     godot_core_module.addIncludePath(gdextension.iface_headers.dirname());
     godot_core_module.addImport("godot", godot_module);
-    lib.root_module.addImport("godot", godot_module);
-    godot_module.addImport("GodotCore", godot_core_module);
-    lib.root_module.addImport("GodotCore", godot_core_module);
 
-    const build_options = b.addOptions();
-    build_options.addOption([]const u8, "precision", precision);
-    build_options.addOption([]const u8, "headers", headers);
-    lib.root_module.addOptions("build_options", build_options);
-    godot_module.addOptions("build_options", build_options);
-    lib.step.dependOn(bindgen.step);
-    b.installArtifact(lib);
+    godot_module.addImport("GodotCore", godot_core_module);
 }
 
 const BindgenOutput = struct {

